.github/workflows/deploy.yml

```diff
@@ -82,9 +84,13 @@ jobs:
         with:
           name: test_results
           path: tests/nightwatch/tests_output
-      - name: Slack notification
-        if: env.SLACK_WEBHOOK_URL != '' && contains(fromJson('["develop", "main", "master", "impl", "val", "prod", "production"]'), env.branch_name) && always ()
-        uses: ravsamhq/notify-slack-action@master
-        with:
-          status: ${{ job.status }}
-          notify_when: "failure"
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
```

Comment:
Our slack notification block will now uses a different github action.  The new github action is docker based, but can be pulled instead of built.  The old action was docker based and had to be built on each deployment.  Fifteen seconds time savings or so.